src/client/styles/objects/columns.scss

```diff
@@ -0,0 +1,6 @@
+.columns {
+    padding: 0px;
+
+    margin-right: 0px !important;  
+    margin-left: 0px !important;
```

Comment:
Is `!important` really necessary?

Reply:
yes, else other scss-variables override the padding